<h1 id="module-3-bioinformatics-and-systems-biology">Module 3. Bioinformatics and Systems Biology</h1>

<p>Computational analysis of biological networks, OMICs data (genomics, transcriptomics, metabolomics, proteomics).  Application of advanced analysis and modeling approaches to study pathways and networks. Emphasis on using existing high throughput data sets alongside newly generated data to analyze and interpret research findings.</p>

<p><strong>N.B.</strong> Please complete this pre-course <a href="http://tinyurl.com/bioinf525-questions"><strong>questionnaire</strong></a> if you have not already done so.</p>

<table>
  <thead>
    <tr>
      <th style="text-align: center"> </th>
      <th style="text-align: center"> </th>
      <th style="text-align: left"> </th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td style="text-align: center"><strong>3.1</strong></td>
      <td style="text-align: center">Lecture</td>
      <td style="text-align: left"><a href="#3.1"><strong>Introduction to systems biology</strong></a></td>
    </tr>
    <tr>
      <td style="text-align: center"> </td>
      <td style="text-align: center">Lab</td>
      <td style="text-align: left"><a href="#3.1">Network analysis for systems biology</a></td>
    </tr>
    <tr>
      <td style="text-align: center"><strong>3.2</strong></td>
      <td style="text-align: center">Lecture</td>
      <td style="text-align: left"><a href="#3.2"><strong>High throughput sequencing methods in systems biology</strong></a></td>
    </tr>
    <tr>
      <td style="text-align: center"> </td>
      <td style="text-align: center">Lab</td>
      <td style="text-align: left"><a href="#3.2">Mapping genetic regulatory networks</a></td>
    </tr>
    <tr>
      <td style="text-align: center"><strong>3.3</strong></td>
      <td style="text-align: center">Lecture</td>
      <td style="text-align: left"><a href="#3.3"><strong>Network modeling for hypothesis testing and generation</strong></a></td>
    </tr>
    <tr>
      <td style="text-align: center"> </td>
      <td style="text-align: center">Lab</td>
      <td style="text-align: left"><a href="#3.3">Modeling and inference of metabolic networks</a></td>
    </tr>
    <tr>
      <td style="text-align: center"><strong>3.4</strong></td>
      <td style="text-align: center">Lecture</td>
      <td style="text-align: left"><a href="#3.4"><strong>Machine learning approaches in systems biology</strong></a></td>
    </tr>
    <tr>
      <td style="text-align: center"> </td>
      <td style="text-align: center">Lab</td>
      <td style="text-align: left"><a href="#3.4">Application of machine learning to biological network analysis</a></td>
    </tr>
  </tbody>
</table>

<p><br /></p>

<hr />
<p><a name="3.1"></a>
<br /></p>

<h4 id="lecture-3-1introduction-to-systems-biology">Lecture (3-1):	<strong>Introduction to systems biology</strong></h4>
<ul>
  <li><strong>Instructor</strong>: 	Dr. Peter Freddolino</li>
  <li><strong>Time</strong>: 		Mar 21 (Tuesday), 2:30 - 4:00 PM</li>
  <li><strong>Topics</strong>:<br />
What are bioinformatics and systems biology?  Major areas of research and application.</li>
  <li><strong>Material</strong>:<br />
<a href="/class-material/BI525W17Lec3.1.pdf">Lecture slides (PDF)</a></li>
</ul>

<p><br /></p>

<h4 id="lab-3-1-network-analysis-for-systems-biology">Lab (3-1): 	<strong>Network analysis for systems biology</strong></h4>
<ul>
  <li><strong>Instructor</strong>: 	Dr. Peter Freddolino</li>
  <li><strong>Time</strong>: 		2:30 – 4:00 PM, Mar 23 (Thursday)</li>
  <li><strong>Topics</strong>:<br />
Introduction to biological network analysis with cytoscape. Mapping of expression data sets onto biological networks. Network design using BioBricks.</li>
  <li><strong>Material</strong>:<br />
<a href="/class-material/lab3.1_w17.pdf">Lab worksheet</a><br />
<a href="/class-material/module3_lab1_solutions.pdf">Lab worksheet with key</a><br />
<a href="https://goo.gl/forms/7El7W1Vw0nHJPUTO2">Muddy point assessment</a></li>
  <li><strong>Homework</strong> (Due 3/30 @ 5:00pm):<br />
<a href="/class-material/BI525W17_HW3.1.pdf">Homework Assignment 1</a></li>
</ul>

<p><br /></p>

<hr />
<p><a name="3.2"></a>
<br /></p>

<h4 id="lecture-3-2-high-throughput-sequencing-methods-in-systems-biology">Lecture (3-2): <strong>High throughput sequencing methods in systems biology</strong></h4>
<ul>
  <li><strong>Instructor</strong>: 	Dr. Peter Freddolino</li>
  <li><strong>Time</strong>: 		Mar 28 (Tuesday), 2:30 - 4:00 PM</li>
  <li><strong>Topics</strong>:<br />
Overview of high throughput sequencing-based methods used to investigate biological networks, along with an introduction to databases and analysis considerations for each.</li>
  <li><strong>Material</strong>:<br />
<a href="/class-material/BI525W17Lec3.2.pdf">Lecture slides (PDF)</a></li>
</ul>

<p><br /></p>

<h4 id="lab-3-2-mapping-genetic-regulatory-networks-using-high-throughput-sequencing">Lab (3-2): 	<strong>Mapping genetic regulatory networks using high-throughput sequencing</strong></h4>
<ul>
  <li><strong>Instructor</strong>: 	Dr. Peter Freddolino</li>
  <li><strong>Time</strong>: 2:30 – 4:00 PM, Mar 30 (Thursday)</li>
  <li><strong>Topics</strong>:<br />
Finding and interpreting RNA-seq and ChIP-seq data sets to study regulatory networks, including identification of differentially expressed genes, location of transcription factor binding sites, and inference of regulatory motifs.</li>
  <li><strong>Material</strong>:<br />
<a href="/class-material/lab3.2_w17.docx">Lab worksheet</a><br />
<a href="/class-material/module3_lab2_solutions.docx">Lab worksheet with key</a><br />
Supporting Files: <a href="/class-material/lab_2_files.zip">lab2_files</a><br />
<a href="https://docs.google.com/a/umich.edu/forms/d/e/1FAIpQLSeJJq3dBvZMCNQuogHrX-719mFChP2H_EtwgYyLO4x5VxYctw/viewform?usp=sf_link">Muddy point assessment</a></li>
  <li><strong>Homework</strong> (Due 4/6 @ 5:00pm):<br />
<a href="/class-material/BI525W17_HW3.2.pdf">Homework Assignment 2</a><br />
<br /></li>
</ul>

<hr />
<p><a name="3.3"></a>
<br /></p>

<h4 id="lecture-3-3-network-modeling-for-hypothesis-testing-and-generation">Lecture (3-3): <strong>Network modeling for hypothesis testing and generation</strong></h4>
<ul>
  <li><strong>Instructor</strong>:   Dr. Peter Freddolino</li>
  <li><strong>Time</strong>:         Apr 4 (Tuesday), 2:30 - 4:00 PM</li>
  <li><strong>Topics</strong>:<br />
Conceptual understanding of kinetic models and constraint-based models of metabolic networks; approaches for translating biochemical schematics into quantitative frameworks.</li>
  <li><strong>Material</strong>:<br />
<a href="/class-material/BI525W17Lec3.3.pdf">Lecture slides (PDF)</a></li>
</ul>

<p><br /></p>

<h4 id="lab-3-3-----modeling-and-inference-of-metabolic-networks">Lab (3-3):     <strong>Modeling and inference of metabolic networks</strong></h4>
<ul>
  <li><strong>Instructor</strong>:   Dr. Peter Freddolino</li>
  <li><strong>Time</strong>:         2:30 – 4:00 PM, Apr 6 (Thursday)</li>
  <li><strong>Topics</strong>:<br />
Students will learn how to represent simple metabolic pathways in SBML, and use the sybil R package to simulate the effects of mutations in a model network.</li>
  <li><strong>Material</strong>:<br />
<a href="/class-material/lab3.3_w17.pdf">Lab worksheet</a><br />
<a href="/class-material/lab_3.3_solutions.docx">Lab worksheet with key</a><br />
<a href="https://goo.gl/forms/VdSt7nMzqxr2233v2">Muddy point assessment</a></li>
  <li><strong>Homework</strong> (Due 4/13 @ 5:00pm):<br />
<a href="/class-material/BI525W17_HW3.3.pdf">Homework Assignment 3</a></li>
</ul>

<p><br /></p>

<hr />
<p><a name="3.4"></a>
<br /></p>

<h4 id="lecture-3-4-machine-learning-approaches-in-systems-biology">Lecture (3-4): <strong>Machine learning approaches in systems biology</strong></h4>
<ul>
  <li><strong>Instructor</strong>: 	Dr. Peter Freddolino</li>
  <li><strong>Time</strong>: 		Apr 11 (Tuesday), 2:30 - 4:00 PM</li>
  <li><strong>Topics</strong>:<br />
Introduction to machine learning and overview of key algorithms (linear classifier, SVM, decision trees and random forests). Existing and potential areas for application of machine learning in the analysis of biological networks.</li>
  <li><strong>Material</strong>:<br />
<a href="/class-material/BI525W17Lec3.4.pdf">Lecture slides (PDF)</a></li>
</ul>

<p><br /></p>

<h4 id="lab-3-4-application-of-machine-learning-to-biological-network-analysis">Lab (3-4): 	<strong>Application of machine learning to biological network analysis</strong></h4>
<ul>
  <li><strong>Instructor</strong>: 	Dr. Peter Freddolino</li>
  <li><strong>Time</strong>: 		2:30 – 4:00 PM, Apr 13 (Thursday)</li>
  <li><strong>Topics</strong>:
Overview of R modules for machine learning; comparison of a variety of approaches for predicting regulatory modules, protein function, and protein-protein interactions.</li>
  <li><strong>Material</strong>:<br />
<a href="/class-material/lab3.4_w17.docx">Lab worksheet</a><br />
<a href="/class-material/lab3.4_w17_solutions.docx">Lab worksheet with key</a><br />
Supporting Files: <a href="/class-material/bioi_mod3_lab4_files.zip">lab4_files</a></li>
  <li><strong>Homework</strong> (Due 4/20 @ 5:00pm):<br />
<a href="/class-material/BI525W17_HW3.4.pdf">Homework Assignment 4</a>, <a href="/class-material/singlecell_dat.csv.gz">singlecell_dat.csv</a></li>
</ul>

<p><br /></p>

<h3 id="reference-material">Reference material</h3>
<!--- files dont exist yet...
[Slides-2.1]()
[Slides-2.2]()
-->

